### PR TITLE
Add more choices of medical equipment for people to supply

### PR DESCRIPTION
**Pull request by issyl0**
https://trello.com/c/CqU89nSq/92-add-more-medical-equipment-to-list

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -166,6 +166,14 @@ en:
               label: "Alcohol hand gel"
             gowns:
               label: "Gowns"
+            face_test_kit:
```
Comment (issyl0) [inline config/locales/en.yml]:
```suggestion
            face_fit_test_kit:
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -166,6 +166,14 @@ en:
...
+              label: "Face fit test kit"
+            face_test_solution:
```
Comment (issyl0) [inline config/locales/en.yml]:
```suggestion
            face_fit_test_solution:
```

File: config/locales/en.yml [hunk 3]
```diff
@@ -166,6 +166,14 @@ en:
...
+              label: "Face fit test solution"
+            swabs:
+              label: "Swabs"
+            detergent:
```
Comment (issyl0) [inline config/locales/en.yml]:
```suggestion
            general_purpose_detergent:
```

**Review by issyl0 — COMMENTED** [review]
Should the keys 100% match the label words?

**Review by alex-ju — APPROVED** [review]

**Pull request merged**